tests/e2e/standard/contacts/contact-summary.wdio-spec.js

```diff
@@ -1,11 +1,8 @@
 const moment = require('moment');
-const constants = require('../../../constants');
 const utils = require('../../../utils');
-const helper = require('../../../helper');
-const commonElements = require('../../../page-objects/protractor/common/common.po.js');
-const loginPage = require('../../../page-objects/protractor/login/login.po.js');
-const contactsPo = require('../../../page-objects/protractor/contacts/contacts.po');
-
+const commonElements = require('../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const contactsPo = require('../../../page-objects/default/contacts/contacts.wdio.page');
 
 describe('Contact summary info', () => {
   const SCRIPT = `
```

Comment:
Is there a way to put this script in another file. For example .../tests/e2e/standard/contacts/config/contact-summary-settings.js